### PR TITLE
replace bar_chart with svg instead of using matplotlib

### PR DESCRIPTION
**Pull request by nichtsfrei**
To have more fine granular control over the bar chart and get an easier
transfer between designer and pheme integration the matplotlib bar chart
gets replaced with an svg template.

**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] [CHANGELOG](https://github.com/greenbone/pheme/blob/master/CHANGELOG.md) Entry
- [ ] Documentation

### CODE REVIEW

File: pheme/templatetags/charts.py [hunk 1]
```diff
@@ -17,26 +17,112 @@
...
+    'High': "#d4003e",
+    'Medium': "#fcb900",
+    'Low': "#7db4d0",
 }
 
 
 register = template.Library()
 
 
+__BAR_ELEMENTS_TEMPLATE = """
```
Comment (y0urself) [inline pheme/templatetags/charts.py:37]:
```suggestion
__BAR_ELEMENT_TEMPLATE = """
```

File: pheme/templatetags/charts.py [hunk 2]
```diff
@@ -17,26 +17,112 @@
...
+    if max_sum == 0:
+        return None
+    bars = ""
+    for i, (key, counts) in enumerate(chart_data.items()):
+        element_x = 0
+        elements = ""
+        for category, count in counts.items():
+            color = title_color.get(category)
+            width = count / max_sum * max_width
+            elements += __BAR_ELEMENTS_TEMPLATE.format(element_x, width, color)
```
Comment (y0urself) [inline pheme/templatetags/charts.py:110]:
```suggestion
            elements += __BAR_ELEMENT_TEMPLATE.format(element_x, width, color)
```

File: pheme/templatetags/charts.py [hunk 3]
```diff
@@ -17,26 +17,112 @@
...
 
 register = template.Library()
 
 
+__BAR_ELEMENTS_TEMPLATE = """
+<rect x="{}" y="17" height="10" width="{}" style="fill: {};"></rect>
+"""
+
+__BAR_TEMPLATE = """
+<!-- transform everytime +44 of previous -->
```
Comment (y0urself) [inline pheme/templatetags/charts.py:42]:
```suggestion
<!-- transform y everytime +44 of previous -->
```

File: pheme/templatetags/charts.py [hunk 4]
```diff
@@ -17,26 +17,112 @@
...
+
+        bars += __BAR_TEMPLATE.format(
+            y=i * bar_jump,
+            key=key,
+            bar_elements=elements,
+            total=sum(counts.values()),
+        )
+
+    svg_chart = __BAR_CHART_TEMPLATE.format(
+        height=len(chart_data.keys()) * bar_jump, bars=bars, legend=""
```
Comment (y0urself) [inline pheme/templatetags/charts.py:121]: `legend` Just a placeholder for later?

**Review by y0urself — CHANGES_REQUESTED** [review]

File: pheme/templatetags/charts.py (hunk 4 above)
Comment (nichtsfrei) [reply pheme/templatetags/charts.py:121]: it's a draft, but, yes :)

File: pheme/templatetags/charts.py [hunk 5]
```diff
@@ -17,25 +17,163 @@
...
+        for i in range(int(max_sum / orientation_basis)):
+            orientation_lines, orientation_labels = __add_orientation(
+                i, orientation_lines, orientation_labels
+            )
+
+        orientation_lines, orientation_labels = __add_orientation(
+            int(max_sum / orientation_basis),
+            orientation_lines,
+            orientation_labels,
+        )
```
Comment (y0urself) [inline pheme/templatetags/charts.py:149]:
```suggestion
        for i in range(int(max_sum / orientation_basis + 1)):
            orientation_lines, orientation_labels = __add_orientation(
                i, orientation_lines, orientation_labels
            )
```

**Review by y0urself — APPROVED** [review]

**Pull request merged**